src/sp/it/pl/main/App.kt

```diff
@@ -489,7 +491,7 @@ class App: Application(), Configurable<Any> {
     fun fetchVMArguments(): List<String> = ManagementFactory.getRuntimeMXBean().inputArguments
 
     /** @return number of instances of this application (including this one) running at this moment */
-    fun getInstances(): Int = VirtualMachine.list().count { it.displayName().contains(AppUtil::class.java.name) }
+    fun getInstances(): Int = VirtualMachine.list().count { it.displayName().contains(App::class.java.name + "Util") }
```

Comment:
No offense, but I wonder how you arrived at the conclusion that this is a good idea. The solution here will break the moment App or AppUtil class is renamed. DisplayName is derived from the name of AppUtil class (and has nothing to do with App class), which means we should refer to AppUtil class, not provide arbitrary string.

We need to push for capturing semantics at compilation time, not the opposite.

Reply:
I know it's weird, but since AppUtil isn't a class anymore, I didn't know how to get its name. It is more of a temporary workaround.

Reply:
I understand now. I'd never figure that out as a reason. It is a Kotlin file, not a Kotlin class. I didn't know Kotlin files compiled to Java classes, but I guess they have to.

Anyway more of a reason to get rid of the file. I suggest moving main() method to App.class.

Reply:
That has already be done. I just check for the package name now, which is the easiest and has the same result.

Reply:
Ok I will check